package/libpiksi/libpiksi/src/settings.c

```diff
@@ -1451,12 +1394,16 @@ static int command_receive_callback(const u8 *data, const size_t length, void *c
 
 static void control_handler(pk_loop_t *loop, void *handle, void *context)
 {
-  control_command_t* cmd_info = (control_command_t*)context;
+  control_command_t *cmd_info = (control_command_t *)context;
 
   u8 data = 0;
   if (pk_endpoint_receive(cmd_info->cmd_ept, command_receive_callback, &data) != 0) {
-    piksi_log(LOG_ERR, "%s: error in %s (%s:%d): %s",
-              __FUNCTION__, "pk_endpoint_receive", __FILE__, __LINE__,
+    piksi_log(LOG_ERR,
+              "%s: error in %s (%s:%d): %s",
+              __FUNCTION__,
+              "pk_endpoint_receive",
+              __FILE__,
+              __LINE__,
               pk_endpoint_strerror());
```

Comment:
Like, why did it decide to indent this function call?  Because it would be over 120 chars?